internal/provisioner/prometheus_operator.go

```diff
@@ -148,7 +145,7 @@ func (p *prometheusOperator) CreateOrUpgrade() error {
 	}
 
 	p.logger.Debugln("CNAME was not provisioned for prometheus")
-	ctx, cancel := context.WithTimeout(context.Background(), time.Duration(120)*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Duration(180)*time.Second)
```

Comment:
Encounter an issue that this helped with?

Reply:
encountered one it _didn't_ help with, and that line snuck in, ugh, thanks for pointing it out